CHANGES.md

```diff
@@ -15,6 +15,7 @@
 ## Current Version
 
  - 3.2.26 (2022-07-20): Minor tweak to NW2d
+ -
```

Comment:
typo?

Reply:
This was the smallest possible change to trigger eyecite benchmark. I'll revert with my next commit.

Reply:
great